yellr-serv/yellrserv/ep_client_post.py

```diff
@@ -0,0 +1,244 @@
+import os
+import json
+import urllib
+import uuid
+
+#import subprocess
+#import magic
+#import mutagen.mp3
+#import mutagen.oggvorbis
+#import mutagen.mp4
+
+from pyramid.view import view_config
+
+import client_utils
+import utils
+from config import system_config
+
+@view_config(route_name='get_local_posts.json')
+def get_local_posts(request):
+
+    result = {'success': False}
+    status_code = 200
+
+    try:
+    #if True:
+
+        success, error_text, language_code, lat, lng, \
+            client = client_utils.register_client(request)
+        if success == False:
+            raise Exception(error_text)
+
+        start = 0
+        count = 50
+        try:
+            if 'start' in reqeusts.GET:
+                start = int(float(request.GET['start']))
+            if 'count' in request.GET:
+                count = int(float(request.GET['count']))
+        except:
```

Comment:
I think it might be a good idea to grep for
```python
except:
    pass
```
because I'm still seeing these pop up without any comments or anything like that.